### ISSUE TITLE
Subtitle font size issue pados 14.2

### ISSUE THREAD

**Issue by IsmaelMiber** [FrontendMasters/ios-issues#48]
I have ipad pro 2020 11 inch, with software version 14.2

The subtitle font size not change when i make it large text from accessibility option in the ipad setting

**Comment by dachev** [comment]
@IsmaelMiber this should be fixed in the latest release

**Comment by IsmaelMiber** [comment]
Unfortunately not fixed yet @dachev

**Comment by dachev** [comment]
@IsmaelMiber Does it help if you leave the course screen and then come back to it again?

**Comment by IsmaelMiber** [comment]
@dachev  This is my setting

![D8583A8D-9F93-421A-8AAF-9126A4740515](https://user-images.githubusercontent.com/16901328/103763263-3ef0a300-5022-11eb-824b-0f40b45b4269.png)
![A46E7B90-3CD2-4120-8846-75A9525F2091](https://user-images.githubusercontent.com/16901328/103763378-67789d00-5022-11eb-99ce-5a62bf1a02c2.png)

**Comment by dachev** [comment]
@IsmaelMiber Does it help if you leave the course screen and then come back to it again?

**Comment by IsmaelMiber** [comment]
Unfortunately, leaving screen not helped

**Comment by dachev** [comment]
Hi @IsmaelMiber. It worked for me what I tested in iOS 10. Can you take a look at this video and tell me if you see something different of you follow the same steps: http://blago.dachev.com/~blago/IMG_4124.mp4

**Comment by IsmaelMiber** [comment]
Hi @dachev , 

Thanks for the video I follow these steps and it work, Why the subtitle font size not related to the subtitles and captions section in the accessibility window of the device, Netflix got the font size from there, I say that because change font size from large text tab change other apps font also because of dynamic font

**Comment by dachev** [comment]
@IsmaelMiber we've implemented custom subtitle rendering which also allows us to make subtitles available offline. We did this by dropping the default system implementation and I guess one side-effect is that we can't read and honor these particular user preferences.

I hope this workaround solves your problem. Given that users are still able to control the font size from a different submenu, and that we haven't received any other complaints, I'm inclined to leave this ticket closed for now until we hear from more people. 

Rest assured that I'm going to keep this in mind the next time we make changes to the player and video steams - we can possibly re-consider the way we show subtitles and go back to a more traditional approach.
